app/views/merchant/discounts/index.html.erb

```diff
@@ -2,9 +2,10 @@
 
 <% @discounts.each do |discount| %>
   <section id="discount-<%= discount.id %>">
-    <p>Name: <%= link_to "#{discount.name}", "/discounts/#{discount.id}/show" %></p>
+    <p>ID: <%= link_to "#{discount.id}", "/merchant/discounts/#{discount.id}" %></p>
+    <p>Name: <%= discount.name %></p>
     <p>Percent Off: <%= discount.percent_off %>%</p>
     <p>Minimum Quantity: <%= discount.min_quantity %></p>
-    <%= link_to "Edit", "/discounts/#{discount.id}/edit" %>
+    <br>
   </section>
```

Comment:
Made the discounts id the link to the show page and not the name. Also removed the edit link on the index page